contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,42 +1,145 @@
 category: base
 doc: |
-  Base class for spectral peaks, their functional form, and support (values).
+  Base class for describing a spectral peak, its functional form, and support values
+  (i.e., the discretization (points) at which the function has been evaluated).
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
-  n_support: |
-    Number of support points
+  n_values: |
+    Number of discretization points at which the functional form of the peak
+    has been evaluated.
+  n_features: |
+    Number of independent data axes to be fitted (for multidimensional fit.)
```

Comment:
Again a different definition. Do you mean multiple data sets, i.e. y1(x), y2(x), etc. (if so, then serial or global fit?), or y(x1, x2, x3, etc.), i.e. multidimensional (or multivariate) fit?

Reply:
This is about multi-dimensional/multi-variate fits. But the docs / symbol names should be better explained, I agree.